tests/php/ORM/TransactionTest.php

```diff
@@ -170,10 +170,8 @@ public function testReadOnlyTransaction()
             DB::get_conn()->transactionRollback();
         }
 
-        DataObject::flush_and_destroy_cache();
-
-        $success = DataObject::get_one(TestObject::class, "\"Title\"='Read only success'");
-        $fail = DataObject::get_one(TestObject::class, "\"Title\"='Read only page failed'");
+        $success = TestObject::get()->find('Title', 'Read only success');
+        $fail = TestObject::get()->find('Title', 'Read only page failed');
```

Comment:
This is about transactions - any reference to caching is completely improper here.